kindtest.sh

```diff
@@ -287,33 +287,31 @@ docker ps
 echo 'Clean up result root...'
 rm -rf "${RESULT_ROOT:?}/*"
 
-if [ "x${maven_profile_name}" = "xkind-sequential" ] ; then
-  parallel_run="false"
-  threads=1
-  echo "Setting the variable parallel_run to [${parallel_run}] for kind-sequential profile"
-fi
-
 # Check for invalid Test Filter/Maven Profile Combination
-
-# Test Filter (all) and Maven Profile is integration-tests
-if [ "x${maven_profile_name}" = "xintegration-tests" ] && 
+if [ "${maven_profile_name}" == "integration-tests" ] && 
    [ "${test_filter}" = "**/It*" ] ; then
-     echo '(ERROR) All tests cannot be run with [integration-tests] profile'
+     echo "(ERROR) All tests cannot be run with [integration-tests] profile"
      exit 0
 fi
 
-# Test Filter is Individual Test Clas(es) and Maven Profile is 
-# not integration-tests
-if [ "x${maven_profile_name}" != "xintegration-tests" ] && 
-   [ "${test_filter}" != "**/It*" ] ; then
-    echo '(ERROR) Individual Test MUST be run with [integration-tests] profile'
-    exit 0
+if [ "${maven_profile_name}" == "kind-sequential" ]; then
+   echo "Overriding the parallel_run to false for kind-sequential profiler"
+   parallel_run=false
+   threads=1
 fi
 
+# If the IT_TEST is not set to "**/It*" (i.e. individual tests are chosen), 
```

Comment:
If IT_TEST is set, integration-test profile is used to run the tests. MAVEN_PROFILE_NAME parameter is ignored.

Reply:
done